docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Namespace extension
+weight: 3
+---
+### Introduction
+
+The Liqo namespace model extends the K8s namespace concepts introducing the support to remote namespaces, "twins" of the local namespace. 
+Those "twin" namespaces map the home cluster namespace on a remote cluster and its associated resources (e.g., services, configmaps). 
+Regarding pod offloading, remote namespace hosts the offloaded pods belonging to the home twin namespace, as they were executed in the home cluster.
+
+### Quick Offloading
+
+Since the very first version, Liqo provides a label-based mechanism to quickly set-up namespace offloading over remote clusters.
+
+To enable it, you should label a target namespace:
+
+```bash
+kubectl label namespace demo-namespace liqo.io/enabled=true
+```
+
+Pods scheduled in the "*demo-namespace*" will be potentially offloaded inside remote clusters. 
+With *quick offloading*, we are selecting all peered clusters suitable for offloading. 
+If you need a fine-grained approach, you can rely on custom offloading and the *namespaceOffloading* resource.
```

Comment:
```suggestion
If you need a fine-grained approach, you can rely on custom offloading and the *NamespaceOffloading* resource.
```
Should be agreed whether CRDs should be typed italic or monospace.

Reply:
As k8s suggests (https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects) we can use italic to define or introduce new terms, and  UpperCamelCase when we refer to an API object already introduced.

Reply:
Seems fine to me